dataprofiler/profilers/profile_builder.py

```diff
@@ -340,7 +340,10 @@ def __init__(self, data, samples_per_update=None, min_true_samples=None,
         self._samples_per_update = samples_per_update
         self._min_true_samples = min_true_samples
         self._profile = dict()
-            
+
+        if isinstance(data, data_readers.text_data.TextData):
+            raise TypeError("Cannot provide text data to Profiler")
```

Comment:
Is this message descriptive enough?

Reply:
should change `text data` to some thing like `TextData object`

Reply:
Resolved